Documentation/debugging/using-your-build.md

```diff
@@ -0,0 +1,70 @@
+Using Your Build
+================
+
+This document explains how to use your CoreFX binaries in a .NET Core console app after you've built the repo.
+
+## Overview
+
+You will create a self-contained .NET Core app via `dotnet publish`, which will include stock images of framework assemblies. Then, you will replace some of those assemblies with ones from your CoreFX build.
+
+## Prerequisites
+
+- .NET Core tools - Install at https://www.microsoft.com/net/core
+
+## Steps
+
+0. Create the .NET Core app you want to use with your CoreFX build.
+
+```sh
+mkdir DotnetLab
+cd DotnetLab
+dotnet new
```

Comment:
It's probably worth mentioning that you should target `netcoreapp2.0` in your "base project" if you intend to copy over the bits from corefx. Technically you want to start with a "base" that is close to what your branch of corefx is producing. For corefx/master that is actually post-netcoreapp2.0, but you should be able to copy master bits on top of 2.0, I think.